xcat-inventory/xcclient/inventory/manager.py

```diff
@@ -369,15 +374,52 @@ def importfromfile(objtypelist, objlist, location,dryrun=None,version=None,updat
             key, value = env.split('=')
             vardict[key] = value
 
+    if 'OBJNAME' in list(jinjavarlist):
+        vardict['OBJNAME']='{{OBJNAME}}'
+
+    oldcwd=os.getcwd()
```

Comment:
Need to check if it is GIT repo first when filling up the GIT related built-in variables,  so no need to run git commands multiple times when the directory is not GIT.

Reply:
fixed